php/razorpay_js/index.html

```diff
@@ -0,0 +1,402 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="UTF-8" />
+	<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+	<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+	<title>Chargebee Razorpay Integeration Sample</title>
+	<style>
+		body{
+			height: 100vh;
+			overflow: hidden;
+		}
+		h1{
+			text-align: center;
+		}
+		.button-container {
+			display: flex;
+			justify-content: center;
+			align-items: center;
+			height: 50%;
+		}
+		.btn-pay{
+			cursor: pointer;
+			background: #003eff;
+			border: none;
+			padding: 12px 15px;
+			border-radius: 5px;
+			color: #e5e5e5;
+			text-transform: uppercase;
+			font-weight: 600;
+    		box-shadow: #0010ff29 2px 2px 6px;
+		}
+		.disable{
+			pointer-events: none;
+			opacity: 0.7;
+		}
+		.dropdown{
+			width: 120px;
+			height: 40px;
+			margin: 1rem;
+			padding: 0rem 1rem;
+			color: #003eff;
+			border: 2px solid #003eff;
+			border-radius: 5px;
+			font-weight: 600;
+		}
+	</style>
+</head>
+
+<body>
+	<h1>Chargebee Razorpay Integeration Sample</h1>
```

Comment:
Correct the spelling for `Integeration` -> `Integration`

Reply:
Done